autoware_dependency_checker/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_dependency_checker</name>
+  <version>0.1.0</version>
+  <description>The autoware_dependency_checker package</description>
+  <maintainer email="yutaka.kondo@youtalk.jp">Yutaka Kondo</maintainer>
+  <license>Apache License 2.0</license>
+  <author email="richard-v@macnica.co.jp">Vincent Richard</author>
+  <author email="masaki.baba@tier4.jp">Masaki Baba</author>
+
```

Comment:
Please insert `ament_cmake` dependency here.

Reply:
Thank you for the suggestion!
I fixed it at https://github.com/autowarefoundation/autoware_tools/pull/89/commits/45c7e3ea0b8b904d120a9981bd8f520c7d263e47